pkgs/development/python-modules/inflate64/default.nix

```
@@ -0,0 +1,37 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  nix-update-script,
+  pythonOlder,
+  setuptools,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "inflate64";
+  version = "1.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-MniCe4A88Aah3yUfPhM3TH0m23eeWjMynMEXibgEvC0=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    homepage = "https://codeberg.org/miurahr/inflate64/";
+    description = "provide Deflater and Inflater class to compress and decompress with Enhanced Deflate compression algorithm";
+    license = lib.licenses.lgpl21Plus;
```

```suggestion
    license = lib.licenses.lgpl21Plus;
    maintainers = [ lib.maintainers.jwillikers ];
```
Same for others. New packages should have a maintainer set.